.github/workflows/python-integrate.yaml

```diff
@@ -14,7 +14,9 @@ jobs:
     name: "[Linux] Python 3.7: Unit Tests"
     runs-on: ubuntu-latest
     outputs:
+      pathChangedAwsLambdaSdk: ${{ steps.pathChanges.outputs.awsLambdaSdk }}
```

Comment:
I think this doesn't belong here (yet)

Reply:
Yes, logically it doesn't make sense here, when this package is not yet part of the project.

This should be included in PR where `aws_lambda_sdk` package is introduced